src/App.css

```diff
@@ -26,6 +26,25 @@
   margin: auto;
 }
 
+.article {
+  font-size: larger;
+  width: 40%;
+  margin: auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.comment {
+  font-size: medium;
+  background-color: rgb(235, 235, 235);
```

Comment:
I'd remove the original css if you're going to do this. For context the bootstrapped css uses the system theme and I'm on dark mode. So my default text colour is white...
![image](https://github.com/user-attachments/assets/7ea8e5ec-f8c8-4934-822b-b8de4263fb96)